workflow/rocoto/tasks.py

```diff
@@ -61,6 +61,8 @@ def __init__(self, app_config: AppConfig, run: str) -> None:
 
         self.n_tiles = 6  # TODO - this needs to be elsewhere
 
+        # DATAROOT is set by prod_envir in ops.  Here, we use `STMP` to construct DATAROOT
+        dataroot_str = f"{self._base.get('STMP')}/RUNDIRS/{self._base.get('PSLOT')}/{self.cdump}.<cyclestr>@Y@m@d@H</cyclestr>"
```

Comment:
@aerorahul This is the problematic line for the CI unit tests.  Change `cdump` to `run`:

```suggestion
        dataroot_str = f"{self._base.get('STMP')}/RUNDIRS/{self._base.get('PSLOT')}/{self.run}.<cyclestr>@Y@m@d@H</cyclestr>"
```